### ISSUE TITLE
jsx-wrap-multilines auto fix makes no sense

### ISSUE THREAD

**Issue by faergeek** [yannickcr/eslint-plugin-react#870]
Hi, great project!

But I have an issue with auto fix for `jsx-wrap-multilines` rule. All it does is just wrapping existing source in parenthesis. So if we had something like this:

``` javascript
const Cmp = () => <div>
  <span />
</div>
```

after auto fix we'll get something like that:

``` javascript
const Cmp = () => (<div>
  <span />
</div>)
```

It would be better, if the output would look something like this:

``` javascript
const Cmp = () => (
  <div>
    <span />
  </div>
)
```

Probably there're some problems with correcting indentation. But in that case would be better to just disable auto fixing support for that rule.

For example, some co-workers on my team use Atom auto fixing. It runs on every save. We just turned that rule on and transitioning to new style, and it started saving code like that, we did not want that, so we just disabled this rule, while fixing it manually :-(

**Comment by e1himself** [comment]
Also it would be great to have a rule that forbids this: 

```jsx
const Cmp = () => (<div>
  <span />
</div>)
```

as technically this is a valid style, but is far from readable.

**Comment by ljharb** [comment]
The paren-wrapping is correct - the indentation rule should be doing the autofixing to the format you specified. Similarly, the indent rule should be able to forbid the latter example if you like.

**Comment by e1himself** [comment]
I don't think *jsx-indent* rule is covering that case. Is that correct?

**Comment by ljharb** [comment]
That might be so (I can't check atm, am mobile). If not, then a PR to include checking that case would be great!

**Comment by faergeek** [comment]
I think prettier works well, doesn't make sense to fix auto fix anymore for me.
